testsuite/tests/kuadrant/gateway/test_basic.py

```diff
@@ -0,0 +1,9 @@
+"""Test for AuthPolicy attached directly to gateway"""
+import pytest
+
+
+@pytest.mark.issue("https://github.com/Kuadrant/kuadrant-operator/pull/287")
+def test_smoke(client):
+    """Test if AuthPolicy attached directly to gateway works"""
+    response = client.get("/get")
+    assert response.status_code == 200
```

Comment:
This does not test that AuthPolicy is working, it would throw 200 even if it didn't, you should do a more complicated AuthPolicy to truly verify the result (like sending it with and without auth)